### ISSUE TITLE
[13] base: Internal list view is lost on wizard after Exception

### ISSUE THREAD

**Issue by sswapnesh** [odoo/odoo#48379]
Impacted versions: 13.0

Steps to reproduce: Select few users --> Change Password --> Keep one of them Empty and Change Password
![screenshot-localhost_8069-2020 03 25-22_26_02](https://user-images.githubusercontent.com/16624719/77563725-a7cdbb00-6ee7-11ea-83e4-24ff678c6b21.png)

Current behavior:
![screenshot-localhost_8069-2020 03 25-22_27_13](https://user-images.githubusercontent.com/16624719/77563863-d9468680-6ee7-11ea-8e84-c85a361be0d1.png)

Expected behavior:
(With Warning)
![screenshot-localhost_8069-2020 03 25-22_28_44](https://user-images.githubusercontent.com/16624719/77563965-02671700-6ee8-11ea-9c8b-ca739e284306.png)

Video/Screenshot link (optional):

**Comment by pedrobaeza** [comment]
It's a bit elaborate and rare that this is an actual problem (as some of your bug reports), but well, let's see if it can be fixed.

**Comment by sswapnesh** [comment]
Maybe, But that's what my Opinion is. (You can go ahead If you find this is (Or any other) expected one.)

**Comment by Yenthe666** [comment]
I don't mean to be rude but lately I've got the feeling you're trying very hard to find issues by doing not-usual operations to create bug reports for. At this point some of the bug reports don't seem to have a lot of impact/benefit anymore. Please don't take it personally, it is just my feeling and maybe @pedrobaeza disagrees with me.

**Comment by sswapnesh** [comment]
Not really trying hard, This bug report is simple state forward. It doesn't include any Developer knowledge to reproduce issue.

(Maybe 5 out of 100 or Counting all (Around 620) There might be 10/20 issue which are not that Important.)

**Comment by pedrobaeza** [comment]
Yes, that's also my feeling and why I put my previous comment. Don't want to discourage you any way, but to focus on usual flows.

**Comment by sswapnesh** [comment]
Update: This is happening at all places where there is no internal view list defined. This works well if there is an internal editable list view.
